foghorn-edge/scripts/bootstrap-edgemanager.sh

```diff
@@ -0,0 +1,12 @@
+yes n | cp -i /fh_vol/fh/tmp/certs/license-ca.crt /fh_vol/fh/certs/ca/ 
+yes n | cp -i /fh_vol/fh/tmp/certs/key.pem /fh_vol/fh/license/edge/ 
```

Comment:
These commands execution requires root privilege? Can you change to non-root user execution?